ledger/src/blockstore.rs

```diff
@@ -2993,6 +2993,7 @@ impl Blockstore {
         lookup_tables.into_par_iter().for_each(|lookup_table_key| {
             bank.get_account(&lookup_table_key)
                 .map(|lookup_table_account| {
+                    add_to_set(&result, &[lookup_table_key]);
```

Comment:
I'm afaid that i'm not bothered to write a unit test. :) after all, this is a very niche dev tool, which should never be used in production code path.